docs/Makefile

```diff
@@ -35,4 +36,4 @@ coverage: Makefile apidoc
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile apidoc
 	cp -r "$(SOURCEDIR)"/* "$(SRCBUILDDIR)"
-	@PYTHONPATH=.. $(SPHINXBUILD) -M $@ "$(SRCBUILDDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+	@PYTHONPATH=$(SRCDIR) $(SPHINXBUILD) -M $@ "$(SRCBUILDDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```

Comment:
I guess this is already how things work, but I don't think we should modify `PYTHONPATH` in this make file.

I usually never manipulate `PYTHONPATH` or `sys.path`, but for people who do, the more common idiom is to add this to `sys.path` in your `conf.py`.

When I have a project where Sphinx requires the project to be on the `PATH`, I usually just make "install the project into the relevant virtualenv" a prerequisite for building the docs.

Reply:
I was modifying `sys.path` in `conf.py` previously, but was asked to stop. I can reintroduce this.

Reply:
Giving it more thought, I suppose this _is_ better than the `sys.path` modification in `conf.py`, because this only affects people building using the `Makefile` and allows us to use more strict build criteria for things like a `tox` environment or in the ReadTheDocs configuration, so I withdraw the objection.